### ISSUE TITLE
Version 5 should be immediately thrown in the bit bucket!!

### ISSUE THREAD

**Issue by rmulder** [joeferraro/MavensMate#565]
I have never before in my life 'bashed' a software product (aside from Microsoft crap) - but I'm gonna do it here today.

Joe - your version 5 of MavensMate - is nothing more than a pile of crap that should be thrown in the trash. Fortunately, my co-worker had been struggling so much trying to get Version 5 to work, that yesterday I sent him a .zip of my MavensMate 4.0.5 folder and my User settings file. If I had not done that - I would have been in a 'world of hurt today.'

For starters, Version 5 is probably one of the 'buggiest' software products I've seen in a long, long time. Secondly, Version 4.x was smart enough to NOT follow symbolic links - which enabled us to have the source code/build process for several static resources right in the project - yet have the symbolic links to node_modules, bower_components, etc. be excluded when doing a 'zip and ship' back up to the SalesForce server. Version 5, however, is 'dumb enough' to want to try and follow them, thus creating a 'static resource' on the order of magnitude of several HUNDRED megabytes - which of course cannot fit within the 5MB limit for static resource files (nor would you want that monster to even be pushed to the cloud).

More importantly, is the fact that you have *forced* users to go back to having the separate MavensMate app. Version 3 was a *pain in the rear* to have to deal with; Version 4 was a 'breath of fresh air' to get away from that complexity, yet with Version 5, you've gone right back to that asinine approach. Why???

The most frustrating thing above all, however, is the fact that I had felt 'safe' from any unwanted/inadvertent update, because in both my User and Default MavensMate settings - I had the following settings:
	//if true, MavensMate will automatically install updates to the MavensMate API, mm
	"mm_auto_install_mm_updates" : false,
Which I *thought* would have protected me from getting this Version 5 piece-of-crap upgrade. You can then imagine my 'shock and horror' to find out after restarting Sublime Text today - that the upgrade happened anyway, without my knowledge and against my express consent.

I feel very, very fortunate that I had made a backup of the important files yesterday, since my colleague had been struggling with this crappy upgrade for over a week now. I had also consulted with a partner and asked them about their experience with Version 5, and got the same response - the 'smart' ones had done everything in their power to prevent the upgrade, whereas the ones not-quite-as-prepared were caught in the upgrade and to this day, continue to fight against the problems caused them by this version.

I don't know about you, but I'm certain that MOST developers using this tool - simply don't have time to 'fart around with' broken/bad/poorly tested software, as 'time is money' and our jobs literally depend on getting things done on time/within budget, and this version has thrown us all into a 'productivity tailspin'. I'm even 'wasting time' writing up this Issue when I should be fixing bugs, but because this caught me completely off guard today - I decided I had to WARN my fellow engineers to find every precaution you can (Time Machine can be your savior too!) to prevent being caught with productivity issues caused by BAD SOFTWARE.

In summary, VERSION 5 IS COMPLETE CRAP AND MUST DIE! LONG LIVE VERSION 4!!!

**Comment by joeferraro** [comment]
Hi @rmulder. I'm sorry you felt the need to express yourself in this manner. It's incredibly hurtful to me and everyone else who contributes to this software. 

I agree v5 has been incredibly challenging for several reasons, which is why at the first sign of problems, I pushed very clear instructions for how to roll back to v4. https://github.com/joeferraro/MavensMate-SublimeText#version-4-no-longer-supported

I am working as we speak to make the v4 branch the official Package Control branch so that folks can go back to using it as they were while I figure out some of the outstanding issues.

**Comment by ralphcallaway** [comment]
@rmulder i know where you're coming from, but i do want to point out a couple things on joe's behalf
- this is free software
- it's open source and is completely maintained by volunteers after they're done working on projects all day
- is 99% the labor of love from joe

while i understanding your frustration, i've done a lot of bashing in my time, you need to consider that this isn't built by a company trying to find a way to charge you $1500/yr/usr per license, and that if you like the software, the best way to lose it completely convince the author that dealing with this is more effort than it's worth

if you want to bash, fine, but don't do it on the issues board, do it to your colleagues, and before you put your opinions out in public, put some skin in the game with at least one pull request, you've made a point about symbolic link behavior not working the way you'd expect, perhaps you might take a look and start there. the community will be a better place for it
